ink-wrapper/src/main.rs

```diff
@@ -18,6 +18,13 @@ struct Args {
         help = "Path to the metadata file to generate a wrapper for."
     )]
     metadata: String,
+
+    #[arg(
+        long,
+        help = "Path to the WASM of the contract relative to the output file. If provided, the WASM will be embedded \
+            in the output file. Making it possible to upload the contract to a chain."
+    )]
+    wasm_path: Option<String>,
```

Comment:
I think there should be an option to use just the `*.contract` file which contains both the metadata and the Wasm code.

Reply:
Sure, but I think that's something to think for for the future, I don't think it makes sense to worry about that in this PR.

Reply:
Then ticket it please, even if it's a small amount of work.

Reply:
:+1: